packages/SwingSet/test/kernel.test.js

```diff
@@ -1817,3 +1817,43 @@ test('reap gc-krefs 12', async t => {
 test('reap gc-krefs overrideNever', async t => {
   await reapGCKrefsTest(t, 12, true);
 });
+
+test('injectEvents', async t => {
+  const endowments = makeKernelEndowments();
+  const { kernelStorage } = endowments;
+  const { kvStore } = kernelStorage;
+  await initializeKernel({}, kernelStorage);
+  const kernel = buildKernel(endowments, {}, {});
+  await kernel.start();
+  {
+    const [start, end] = JSON.parse(kvStore.get('acceptanceQueue'));
+    t.deepEqual([start, end], [1, 1]);
+  }
```

Comment:
I'll add the comment, but the real question is whether to expose the `kernelKeeper` object itself, maybe through some `kernel` debug/test method that doesn't exist yet. It would help with the stats checks at the end of this test tool.